### PR TITLE
Feature/rkm66 68 nav bar

### PR DESCRIPTION
**Pull request by PwFaze**
## Change made

-   [ ] Bug fixes
-   [x] New features
-   [ ] Breaking changes

## Describe what you have done

-

### New Features

-

### Fix

-

### Others

-

### CODE REVIEW

File: src/pages/profile.tsx [hunk 1]
```diff
@@ -1,4 +1,14 @@
 // import WaitForBaanselection from "@/components/Profile/WaitForBaan";
+import PhaseTwoNavbar from '@/components/PhaseTwoNavbar';
 import BaanSelection from '@/components/Profile/BaanSelection';
 
+const Test = () => {
```
Comment (shalluv) [inline src/pages/profile.tsx]: Please rename this.

File: src/components/PhaseTwoNavbar.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                </Link>
+            </div>
+            <div className="relative flex h-full gap-8">
+                {/* wait for link to be added */}
+                <Link href="/">
+                    <NewspaperIcon
+                        className={`h-full w-12 hover:text-amber-400 ${
+                            router.pathname === '/article-page'
+                                ? 'text-amber-400'
+                                : 'hover:text-amber-400'
```
Comment (shalluv) [inline src/components/PhaseTwoNavbar.tsx]: The code `hover:text-amber-400` already exists, so when `router.pathname != '/article-page'`, it has no effect. You can remove it if I am correct.

File: src/components/PhaseTwoNavbar.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                        }`}
+                    />
+                </Link>
+                {/* wait for link to be added */}
+                <Link href="/">
+                    <BellIcon
+                        className={`h-full w-12 hover:text-amber-400 ${
+                            router.pathname === '/WAITTOADD'
+                                ? 'text-amber-400'
+                                : 'hover:text-amber-400'
```
Comment (shalluv) [inline src/components/PhaseTwoNavbar.tsx]: Same as above.

File: src/components/PhaseTwoNavbar.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                        }`}
+                    />
+                </Link>
+                {/* wait for link to be added */}
+                <Link href="/">
+                    <HomeIcon
+                        className={`h-full w-12 hover:text-amber-400 ${
+                            router.pathname === '/'
+                                ? 'text-amber-400'
+                                : 'hover:text-amber-400'
```
Comment (shalluv) [inline src/components/PhaseTwoNavbar.tsx]: Same as above.

File: src/components/PhaseTwoNavbar.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+import Link from 'next/link';
+import { HomeIcon, NewspaperIcon, BellIcon } from '@heroicons/react/24/solid';
+import Favicon from '@/public/images/favicon.svg';
+import { useRouter } from 'next/router';
+
+function PhaseTwoNavbar() {
+    const router = useRouter();
+
+    return (
+        <nav className="sticky top-0 z-40 flex h-11 w-full justify-between bg-black/50 px-2 py-2 backdrop-blur-sm md:px-5 lg:h-20 lg:px-10 lg:py-4">
```
Comment (shalluv) [inline src/components/PhaseTwoNavbar.tsx]: Instead of using `px-2` and `py-2`, you can simply use `p-2`.

**Review by shalluv — CHANGES_REQUESTED** [review]

File: src/components/PhaseTwoNavbar.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+                        className={`h-full w-12 hover:text-amber-400 ${
+                            router.pathname === '/article-page'
+                                ? 'text-amber-400'
+                                : ''
+                        }`}
+                    />
+                </Link>
+                {/* wait for link to be added */}
+                <Link href="/">
+                    <BellIcon
```
Comment (boomchanotai) [inline src/components/PhaseTwoNavbar.tsx:35]: Does it have this button on design?

**Review by boomchanotai — CHANGES_REQUESTED** [review]
Super cool!

**Review by isd-team-sgcu — APPROVED** [review]
Well Done !

**Review by boomchanotai — APPROVED** [review]
LGTM!

**Review by shalluv — APPROVED** [review]
Boom told me to approve, and I trust Boom with my life 😘

**Pull request merged**